src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/RestApiPrivilegesEvaluator.java

```diff
@@ -216,6 +216,10 @@ public String checkAccessPermissions(RestRequest request, Endpoint endpoint) thr
 			logger.debug("Checking admin access for endpoint {}, path {} and method {}", endpoint.name(),  request.path(), request.method().name());
 		}
 
+		if (endpoint == Endpoint.ACCOUNT) {
+			return null;
```

Comment:
why we are returning `null`?  Can you add comments for this?
I guess we are checking if user has an access to Account endpoint. And each user will have an access irrespective of type of permission they have.

Reply:
The function should null to grant permission (similar to returning 0) for success. Subsequent checks are done the same way. I have added for this. Will add comments too